atom/browser/api/atom_api_session.cc

```diff
@@ -142,6 +142,16 @@ void SetUserAgentInIO(scoped_refptr<net::URLRequestContextGetter> getter,
           user_agent));
 }
 
+void ResolvePromise(atom::util::Promise promise) {
```

Comment:
These resolve and reject helpers are spread across files and it seems like a pattern, I would prefer them as static helper methods available from `atom/common/promise_util.h`. Can help cleanup some recent code changes.

Reply:
will do!